### ISSUE TITLE
Tensorflow 기반의 LSTM 학습 모델 설계

### ISSUE THREAD

**Issue by bonopi07** [bonopi07/2018-1_advML_project#2]
1. LSTM 모델 디자인
 - layer를 얼마나 쌓을 것인가?
 - model의 input, output shape를 어떻게 디자인할 것인가?

2. 쉽게 실험할 수 있도록 변경가능한 값들은 config file로 관리하기 (.ini 확장자 파일 사용 예정)

3. 어떤 loss function, optimizer를 사용할 것인가?

4. OOP level로 구현해보기

**Comment by bonopi07** [comment]
1. Layer : 1층 -> 3층(추후 수정)
1. input/output shape: (input)char-level vector, (output) probability
2. success
3. loss function: softmax_cross entropy, optimizer: Adam
4. success

**Comment by bonopi07** [comment]
실험 진행
- Layer: 3층의 LSTM으로 구성
- input: char별로 one-hot encoding을 적용한 vector
- output: 1~5점의 정수 평점에 대한 softmax vector
- loss function: softmax_cross_entropy, optimizer: Adam
=> 정확도: 41.01%

**Comment by bonopi07** [comment]
실험 진행
- Layer: 3층의 LSTM으로 구성
- input: char별로 one-hot encoding을 적용한 vector
- output: 1~5점의 정수 평점에 대한 softmax vector
- loss function: softmax_cross_entropy, optimizer: Adam
=> 정확도: 41.01%
